lib/taurus/qt/qtgui/panel/taurusmodelchooser.py

```diff
@@ -53,7 +53,7 @@ class TaurusModelSelector(Qt.QTabWidget):
     # TODO add action to add new TaurusModelSelectorItem
     # TODO add mechanism to allow manual plugin activation
     #      (instead of relying on installation)
-    modelsAdded = Qt.pyqtSignal('QStringList')
+    modelsAdded = Qt.pyqtSignal(list)
```

Comment:
If "QStringList" is not working, I think that it is better to use `taurus.external.qt.compat.PY_OBJECT` instead of `list`